src/bindings/lua/flux-lua.c

```diff
@@ -1096,53 +1112,45 @@ static int l_flux_reactor_stop_error (lua_State *L)
-    { "recvmsg",         l_flux_recvmsg     },
-    { "rpc",             l_flux_rpc         },
-    { "sendevent",       l_flux_send_event  },
-    { "recv_event",      l_flux_recv_event },
-    { "subscribe",       l_flux_subscribe   },
-    { "unsubscribe",     l_flux_unsubscribe },
-    { "getattr",         l_flux_getattr     },
-    { "msghandler",      l_msghandler_add    },
-    { "statwatcher",     l_stat_watcher_add  },
-    { "timer",           l_timeout_handler_add },
-    { "reactor",         l_flux_reactor_start },
-    { "reactor_stop",    l_flux_reactor_stop },
-    { "reactor_stop_error",
-                         l_flux_reactor_stop_error },
-    { NULL,              NULL               }
-};
-
-static const struct luaL_Reg msghandler_methods [] = {
-    { "__index",         l_msghandler_index    },
-    { "__newindex",      l_msghandler_newindex },
-    { "remove",          l_msghandler_remove   },
-    { NULL,              NULL                  }
-};
-
-static const struct luaL_Reg watcher_methods [] = {
-    { "__gc",            l_watcher_destroy  },
-    { "__index",         l_watcher_index    },
-    { "remove",          l_watcher_remove   },
-    { NULL,              NULL                  }
-};
-
-#define FLUX_CONSTANT_SET(L, name) do { \
-  lua_pushlstring(L, #name, sizeof(#name)-1); \
-  lua_pushnumber(L, FLUX_ ## name); \
-  lua_settable(L, -3); \
-} while (0);
-
+static const struct luaL_Reg flux_functions[] = {{"new", l_flux_new}, {NULL, NULL}};
+
+static const struct luaL_Reg flux_methods[] = {{"__gc", l_flux_destroy},
+                                               {"__index", l_flux_index},
+                                               {"send", l_flux_send},
+                                               {"recv", l_flux_recv},
+                                               {"recvmsg", l_flux_recvmsg},
+                                               {"rpc", l_flux_rpc},
+                                               {"sendevent", l_flux_send_event},
+                                               {"recv_event", l_flux_recv_event},
+                                               {"subscribe", l_flux_subscribe},
+                                               {"unsubscribe", l_flux_unsubscribe},
+                                               {"getattr", l_flux_getattr},
+                                               {"msghandler", l_msghandler_add},
+                                               {"statwatcher", l_stat_watcher_add},
+                                               {"timer", l_timeout_handler_add},
+                                               {"reactor", l_flux_reactor_start},
+                                               {"reactor_stop", l_flux_reactor_stop},
+                                               {"reactor_stop_error",
+                                                l_flux_reactor_stop_error},
+                                               {NULL, NULL}};
+
```

Comment:
Perhaps this is the same case @chu11 already pointed out, but anything we can do here? clang-format's result is *way* less readable IMO